pkg/k8s/resource_ctors.go

```diff
@@ -107,6 +108,17 @@ func CiliumIdentityResource(lc hive.Lifecycle, cs client.Clientset, opts ...func
 	return resource.New[*cilium_api_v2.CiliumIdentity](lc, lw, resource.WithMetric("CiliumIdentityList")), nil
 }
 
+func NetworkPolicyResource(lc hive.Lifecycle, cs client.Clientset, opts ...func(*metav1.ListOptions)) (resource.Resource[*slim_networkingv1.NetworkPolicy], error) {
+	if !cs.IsEnabled() {
+		return nil, nil
+	}
+	lw := utils.ListerWatcherWithModifiers(
+		utils.ListerWatcherFromTyped[*slim_networkingv1.NetworkPolicyList](cs.Slim().NetworkingV1().NetworkPolicies("")),
+		opts...,
+	)
+	return resource.New[*slim_networkingv1.NetworkPolicy](lc, lw, resource.WithMetric("NetworkPolicyList")), nil
```

Comment:
Metric should be just "NetworkPolicy"

Reply:
Ack - should the above `CiliumIdentityList` then also be just `CiliumIdentity`?